lib/projectLink.js

```diff
@@ -9,6 +9,10 @@ class ProjectLink {
     await db.addLink(this.project, this.label, this.url);
   }
 
+  async delete(db) {
+    await db.deleteLink(this.project, this.label, this.url);
```

Comment:
The final parameter (url) is unnecessary here as it is not actually used by the `deleteLink` method.